spec/support/features/page_helpers.rb

```diff
@@ -0,0 +1,153 @@
+# frozen_string_literal: true
+
+module Features
+  module PageHelpers
```

Comment:
Can we stop PR bloat and start slicing it to smaller chunks please.
Otherwise we end up with forked styles

Reply:
I appreaciate this PR is bigger than it should be (won't happen often), but I was an eager beaver to refactor the features spec while I was in the area.